libs/labelbox/src/labelbox/client.py

```diff
@@ -881,13 +815,97 @@ def create_model_evaluation_project(self,
             dataset_name_or_id = dataset_name
 
         kwargs["media_type"] = MediaType.Conversational
-        kwargs["ontology_kind"] = OntologyKind.ModelEvaluation
         kwargs["dataset_name_or_id"] = dataset_name_or_id
         kwargs["append_to_existing_dataset"] = append_to_existing_dataset
         kwargs["data_row_count"] = data_row_count
+        kwargs["editor_task_type"] = EditorTaskType.ModelChatEvaluation.value
+
+        return self._create_project(**kwargs)
+
+    def create_offline_model_evaluation_project(self, **kwargs) -> Project:
+        kwargs["media_type"] = MediaType.Conversational
```

Comment:
Re: typed arguments, see my proposal above https://github.com/Labelbox/labelbox-python/pull/1682#discussion_r1645233862
I will add a docstring (forgot to add it). Since these arguments are not user-facing, I propose to clarify them as code comments

Reply:
it is user facing, but the user is not expected to pass in / set those args